packages/api/cli/package.json

```diff
@@ -12,9 +12,11 @@
   },
   "devDependencies": {
     "@malept/cross-spawn-promise": "^2.0.0",
+    "@types/update-notifier": "6.0.8",
     "chai": "^4.3.3",
     "chai-as-promised": "^7.0.0",
-    "mocha": "^9.0.1"
+    "mocha": "^9.0.1",
+    "update-notifier": "5.1.0"
```

Comment:
also, can we not dynamically import the ESM module into CommonJS? Would be nice to have v6. v7 seems out of the question for now because we still support Node 16 in Forge.

`await import('update-notifier')`

Reply:
I tried to use the dynamic import() for v6 but kept getting the same ESM errors.